lib/mixlib/install/backend/base.rb

```diff
@@ -189,6 +189,15 @@ def windows_artifact_fixup!(artifacts)
             end
           end
 
+          # We currently only support appx packages for 2016nano
+          artifacts.delete_if do |artifact|
+            artifact.url.end_with?(".appx") unless %w{2016nano}.include?(artifact.platform_version)
```

Comment:
Let's move the list of nano platform versions up in a `SUPPORTED_WINDOWS_NANO_VERSIONS` constant like we have for `SUPPORTED_WINDOWS_DESKTOP_VERSIONS`.